11_developer_primer/3_developing_for_dynamo/advanced-dynamo-node-customisation.md

```diff
@@ -180,7 +180,181 @@ public static void ProcessFile(string filePath)
 
 In this example:
 
-* **Condition**: If the file path doesn�t end with �.csv�.
+* **Condition**: If the file path doesnt end with .csv.
```

Comment:
Thank you for correcting this instance, would you sanity check again over all? I did see the following for the already merged content:
![image](https://github.com/user-attachments/assets/527b4c65-d584-42d9-8718-18f1b90319c7)

Reply:
I've completed a find and replace for these symbols, so none should still be present in this file